examples/sinkhorn_loss_functional/main.py

```diff
@@ -47,11 +48,19 @@
     print(M)
 
     # Compute the transport matrix between each pair of sets in the minibatch with default parameters
-    P = sinkhorn(a, b, M, 1e-3)
-
+    P = sinkhorn(a, b, M, 1e-3, max_iters=5000, stop_thresh=1e-12)
```

Comment:
You made `max_iters` 500 in the other example. Maybe keep this one 500 as well.

Reply:
It is already changed in the second commit.